docs/howto/fixing-conflicts.md

```diff
@@ -1,3 +1,43 @@
-# Fixing conflicts in Mondo
+# Fixing merge conflicts
+
+This [video](https://drive.google.com/file/d/1DqYiXEdkLCVji55FmFlA9sPmfLouEqrj/view?usp=sharing) illustrates an example of fixing a merge conflict in the mondo ontology.
+
+
+
+**Instructions:**
+
+1. If a merge conflict error appears in your Github.com pull request after commiting a change, open GitHub Desktop and select the corresponding repository from the "Current Repository" button.
+
+2. With the repository selected, click the "Fetch origin" button to fetch the most up-to-date version of the repository.
+3. Click the "Current Branch" button and select the branch with the merge conflict.
+
+4. From the menu bar, select Branch > "Update from master".
+
+5. A message indicating the file with a conflict should appear along with the option to open the file (owl or obo file) in a text/code editor, such as Sublime Text. Click the button to open the file. 
+*See "Optional Reserialisation" below before executing the following steps to potentially make the fix easier to resolve.*
+
+6. Search the file for conflict markings ( <<<<<<< ======= >>>>>>> ).
+
+7. Make edits to resolve the conflict, e.g., arrange terms in the correct order.
+
+8. Remove the conflict markings.
+
+9. Save the file.
+
+10. Open the file in Protégé. If prompted, do not reload any previously opened file. Open as a new file.
+
+11. Check that the terms involved in the conflict appear OK, i.e., have no obvious errors.
+
+12. Save the file in Protégé using File > 'Save as...' from the menu bar and replace the ontology edit file, e.g., mondo-edit.obo
+
+13. Return to GitHub Desktop and confirm the conflicts are now resolved. Click the "Continue Merge" button and then the "Push origin" button.
+
+14. Return to Github.com and allow the QC queries to rerun.
+
+15. The conflict should be resolved and the branch allowed to be merged.
+
+
+*Optional Reserialisation*
+
+If the merge involves a serialisation issue, adapting and executing [the instructions here](https://github.com/obophenotype/cell-ontology/blob/master/docs/resolving_merge_conflicts.md) for your repository could make the conflict easier to resolve.
```

Comment:
personally I would rather you not link to the CL doc -> probably better to just write out the command. My ideal situation would be that CL no longer needs that doc after this is done.

Reply:
I'm a bit unclear on this normalize_src command. If the editor follows the instructions as presented in the video and outlined in the steps, wouldn't that be sufficient, making the normalize_src command unnecessary? 
Can you explain what normalize_src does that would not be covered in these instructions? If we can avoid editors using docker and terminal to resolve conflicts, that would make the process less complicated.